store/src/commonMain/kotlin/org/mobilenativefoundation/store/store5/impl/FetcherController.kt

```diff
@@ -55,7 +56,16 @@ internal class FetcherController<Key : Any, Network : Any, Output : Any, Local :
      */
     private val sourceOfTruth: SourceOfTruthWithBarrier<Key, Network, Output, Local>?,
 
-    private val converter: Converter<Network, Output, Local>? = null
+    private val converter: Converter<Network, Local, Output> = object : Converter<Network, Local, Output> {
```

Comment:
Can you add a comment explaining why it is safe to use this object as the converter?

Reply:
This is due to a bit of tangling between store and mutable store.  In store, there is no concept of local, the network type is always the same as what is going into the source of truth. This default parameter keeps us from having to pass in that dummy everywhere. The second function `fromOutputToLocal` will never be called, I'll throw an exception from there.